packages/internal-scripts/src/treemirror.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import fastGlob from 'fast-glob';
+import * as fs from 'fs/promises';
+import * as pathlib from 'path';
+
+const unreachable = (x: never) => x;
+
+const main = async () => {
+  const [, , mode, root, dest, ...globs] = process.argv;
+  if (
+    !(mode === 'copy' || mode === 'symlink') ||
```

Comment:
is symlink used anywhere currently?

Reply:
Removed. I was trying out both approaches, because a symlink works better for a cross-package `tsc --watch` workflow. But we must have copies for npm publish because npm ignores symlinks. We could do both depending on workflow, but it was a bit more complicated, plus was slightly worried about somehow accidentally having the wrong type of file there for publish.